packages/maskbook/src/_locales/en/messages.json

```diff
@@ -326,7 +334,7 @@
     "plugin_wallet_connect_to_walletconnect_on_web": "Scan with WalletConnect to connect",
     "plugin_wallet_connect_to_walletconnect_on_app": "Connect by WalletConnect",
     "plugin_wallet_connect_more": "More",
-    "plugin_wallet_connect_more_description": "Comming soon…",
+    "plugin_wallet_connect_more_description": "Add custom network",
```

Comment:
plz add an new i18n key.....

Reply:
Will reset the same field in other locate files.